workflow/ush/detect_machine.sh

```diff
@@ -79,3 +80,5 @@ if [[ "${MACHINE}" == "UNKNOWN" ]]; then
     echo WARNING: UNKNOWN PLATFORM 1>&2
   fi
 fi
+
+export MACHINE=${MACHINE}
```

Comment:
Why we need line 84? It was already exported at line 3.

Reply:
removed this line.